inappmessaging/USERGUIDE.md

```diff
@@ -347,6 +354,12 @@ Documents targeting Product Managers:
 
 ## <a name="changelog"></a> Changelog
 
+### 6.0.0 (in-progress)
+* SDKCF-4151: **Breaking Changes:**
+  - Renamed method for providing access token in `UserInfoProvider` interface class from `provideRaeToken` to `provideAccessToken`.
+  - Removed `provideRakutenId` method for Rakuten Id in `UserInfoProvider` interface class from `provideRaeToken` to `provideAccessToken`. Please use `provideUserId` for specific user targeting.
```

Comment:
'... from `provideRaeToken` to `provideAccessToken' - does that belong in this sentence?